libpod/container_internal_linux.go

```diff
@@ -805,3 +805,9 @@ func (c *Container) makePlatformMtabLink(etcInTheContainerFd, rootUID, rootGID i
 func (c *Container) getPlatformRunPath() (string, error) {
 	return "/run", nil
 }
+
+func (c *Container) addMaskedPaths(g *generate.Generator) {
+	if !c.config.Privileged {
```

Comment:
should we also add `&& len(g.Config.Linux.MaskedPaths) != 0` as check here? if `umask=all` is set it would mean we do not add the path which seems more desirable.

Reply:
I agree with @Luap99, it'd be better to have a way to not mask it.

Otherwise we have no way to have an unmasked `/sys` in the container and I think that breaks mounting `/sys` in a nested container